### PR TITLE
Add mailchimp_ecommerce_get_cart()

### PR DESCRIPTION
**Pull request by aaronbowwwls**
This fixes issue #144

### CODE REVIEW

File: mailchimp_ecommerce.module [hunk 1]
```diff
@@ -699,6 +699,40 @@ function mailchimp_ecommerce_get_order($order_id) {
   return NULL;
 }
 
+/**
+ * Gets a cart from the current MailChimp store.
+ *
+ * @param string $cart_id
+ *   The cart ID.
+ *
+ * @return object
```
Comment (taz77) [inline mailchimp_ecommerce.module]: The return is mixed. An object or NULL can be returned.

**Review by taz77 — APPROVED** [review]
Docblock could be adjusted for return.

File: mailchimp_ecommerce.module (hunk 1 above)
Comment (aaronbowwwls) [reply mailchimp_ecommerce.module]: Good call.

**Comment by aaronbowwwls** [comment]
Issue #146 is fixed by commit fc8d53428274d1f76b88fd3cdafe6caac048ca71.

**Pull request merged**